test/integration/scheduler_perf/create.go

```diff
@@ -40,7 +45,13 @@ type createAny struct {
 	// Namespace the object should be created in. Must be empty for cluster-scoped objects.
 	Namespace string
 	// Path to spec file describing the object to create.
+	// This will be processed with text/template.
+	// .Index will be in the range [0, Count-1] when creating
+	// more than one object. .Count is the total number of objects.
 	TemplatePath string
+	// Count determines how often to create the object. Defaults to 1 if unset.
```

Comment:
How ofter or how many?

Reply:
How many, but that is also "how often" because each object gets created with one call :smile: 

I've changed to "how many" because that is what matters.